packages/tc-schema-sdk/data-accessors/relationship-types.js

```diff
@@ -0,0 +1,32 @@
+const type = require('./type');
+
+module.exports = {
+	// todo move to object rest parameters for options when upgrading node
+	accessor(options = {}) {
```

Comment:
I would add a comment at the top of this function explaining that it gets _all_ relationshipTypes - a mixture of user defined and automatically generated